lib/hca/enrollment_eligibility/service.rb

```diff
@@ -28,32 +28,31 @@ class Service < Common::Client::Base
         'groupNumber' => 'insuranceGroupCode'
       }.freeze
 
-      MARITAL_STATUSES = %w[
-        Married
-        Never Married
-        Separated
-        Widowed
-        Divorced
+      MARITAL_STATUSES = [
+        'Married',
+        'Never Married',
+        'Separated',
+        'Widowed',
+        'Divorced'
       ].freeze
 
       MEDICARE = 'Medicare'
 
-      def get_ezr_data(icn)
+      def get_ezr_prefill_data(icn)
         response = with_monitoring do
           lookup_user_req(icn)
         end
 
+        financial_info = parse_financial_info(response)
```

Comment:
Added financial info to the prefill hash. It wasn't a ton of work, so I included it in this PR.